eip/EIPS/eip-aa-rules.md

```diff
@@ -210,7 +210,8 @@ The permanent storage access with `SLOAD` and `SSTORE` instructions within each
     * **[STO-022]**  There is an `initCode` and the `factory` contract is staked.
 * If the entity (`paymaster`, `factory`) is staked, then it is also allowed:
     * **[STO-031]** Access the entity's own storage.
-    * **[STO-032]** Access a storage slot in any non-entity contract that is associated with the entity.
+    * **[STO-032]** Write Access to storage slots that is associated with the entity, in any non-entity contract.
```

Comment:
"Full access" or "Read/Write access", since the next line says "Read-only" rather than "Read" so it's not exactly complementing.